app/views/player.js

```diff
@@ -64,14 +65,25 @@ const Player = Mn.LayoutView.extend({
                     break;
             }
         });
+        $(document).mousemove((e) => {
```

Comment:
I'm not especially fan of listening events that comes from `document` inside a component, because it can have side-effects. I think we can leave it at this time, but don't hesitate to listen to `document` from the `appLayout` and broadcast the event to the child components when needed.